index.js

```diff
@@ -27,11 +27,16 @@ module.exports = function(content) {
     return callback(null, content);
   } else {
     var plugins = [];
-    plugins.push(imageminGifsicle(options.gifsicle));
-    plugins.push(imageminMozjpeg(options.mozjpeg));
-    plugins.push(imageminSvgo(options.svgo));
-    plugins.push(imageminPngquant(options.pngquant));
-    plugins.push(imageminOptipng(options.optipng));
+    if(options.gifsicle.enabled !== false)
```

Comment:
This will fail if options.gifsicle is undefined or false. Same for other options.